### ISSUE TITLE
Save user_answers using Eloquent instead query builder (DB::Facade)

### ISSUE THREAD

**Issue by bytech-ca** [eclectic-bytech/webcourses-ngx-v4#266]
`UserAnswerController.php` on line 60 is saving answers using DB::Facade (query builder). I could have sworn we had this one already updated, but cannot find a related pull request, nor branch (a missing branch might be simply due to cleanup, before a full merge was done.) @SirL3X0N Do check your local branches for the updated code before starting on this one...

Either I am dreaming that this was done, or it was bundled with another issue, and somehow missed in a merge (probably when another branch was created.)

Note the comment in the code on what and how needs to be done:
```
// Using DB::table is a sure sign we're not using Eloquent.  
// An example of replacing query builder (DB::table) with Eloquent can be found here:  
// https://github.com/eclectic-bytech/webcourses-ngx-v4/commit/007f61ace77348af3d56c6e3a1c705599e406485  
```

**Comment by SirL3X0N** [comment]
No no I did this one the branch is #255

**Comment by SirL3X0N** [comment]
You merged it into dev branch I think ¯\_(ツ)_/¯

**Comment by bytech-ca** [comment]
Hm, branch 255 was cleaned out, and issue 251 doesn't have the patch for this item. Dev was merged into master earlier this week, and the patch isn't there either. If the solution is on branch 255, hopefully you have a local copy you can re-upload. Otherwise, I am not seeing the patch anywhere in the repo, and this might need be redone. :(

**Comment by SirL3X0N** [comment]
Luck has it I do. There was a pull request made for the branch tho and it was merged into dev by you so I'm lost as to what happened
